docs/approach.rst

```diff
@@ -344,57 +365,58 @@ The blue and red lines show the predicted values for the :math:`S_0` and
+``tedana`` includes three options `tedana_orig`, `meica` and `minimal`
+(which uses hardcoded thresholds applied to each of the metrics).
+These decision trees are detailed in :doc:`/included_decision_trees`.
 
 Components that are classified as noise are projected out of the optimally combined data,
 yielding a denoised timeseries, which is saved as **desc-denoised_bold.nii.gz**.
 
 .. image:: /_static/a15_denoised_data_timeseries.png
 
-.. _These decision trees are detailed here: included_decision_trees.html
 
 *******************************
 Manual classification with RICA
 *******************************
 
-``RICA`` is a tool for manual ICA classification. Once the .tsv file containing the result of
-manual component classification is obtained, it is necessary to `re-run the tedana workflow`_
-passing the manual_classification.tsv file with the --ctab option. To save the output correctly,
-make sure that the output directory does not coincide with the input directory. See `this example`_
-presented at MRITogether 2022 for a hands-on tutorial.
+``RICA`` is a tool for manual ICA classification.
+Once the .tsv file containing the result of manual component classification is obtained,
+it is necessary to re-run the tedana workflow
+(see :ref:`usage:running the ica_reclassify workflow`)
+passing the ``manual_classification.tsv`` file with the ``--ctab`` option.
+To save the output correctly,
+make sure that the output directory does not coincide with the input directory.
+See `this example`_ presented at MRITogether 2022 for a hands-on tutorial.
 
-.. _re-run the tedana workflow: https://tedana.readthedocs.io/en/stable/usage.html#Arguments%20for%20Rerunning%20the%20Workflow
 .. _this example: https://www.youtube.com/live/P4cV-sGeltk?feature=share&t=1347
 
 
 *********************************************
 Removal of spatially diffuse noise (optional)
 *********************************************
 
-:func:`tedana.gscontrol.gscontrol_raw`, :func:`tedana.gscontrol.gscontrol_mmix`
-
-Due to the constraints of ICA, TEDICA is able to identify and remove spatially
-localized noise components, but it cannot identify components that are spread
-out throughout the whole brain. See `Power et al. (2018)`_ for more information
-about this issue.
-One of several post-processing strategies may be applied to the ME-DN or ME-HK
-datasets in order to remove spatially diffuse (ostensibly respiration-related)
-noise.
-Methods which have been employed in the past include global signal
-regression (GSR), minimum image regression (MIR), anatomical CompCor, Go Decomposition (GODEC), and
-robust PCA.
+:func:`tedana.gscontrol.gscontrol_raw`, :func:`tedana.gscontrol.minimum_image_regression`
+
+Due to the constraints of spatial ICA,
+TEDICA is able to identify and remove spatially localized noise components,
+but it cannot identify components that are spread out throughout the whole brain.
```

Comment:
I've seen this logic before, but it still makes no sense to me. ICA complete ARE spatially diffuse and ICA components can also be minimally clustered. In fact, the lack of clustering is one of our metrics to reject components. Additionally, there is little evidence besides one replicated paper that GSR or the other methods actually address this issue.
If it were up to me, I'd cut this entire section, but I'll defer if others disagree.

Reply:
It's a point that comes up in most spatial vs. temporal ICA papers I've read. I haven't directly tested it or anything, but I haven't seen anyone argue against it in a way I could cite.